qhub/cli/deploy.py

```diff
@@ -15,7 +15,6 @@ def create_deploy_subcommand(subparser):
     subparser.add_argument(
         "-c", "--config", help="qhub configuration yaml file", required=True
     )
-    subparser.add_argument("-i", "--input", help="input directory")
```

Comment:
Don't recall why this change?

Reply:
You used to be able to supply an alternative cookie-cutter template location, but we don't believe it was useful or in use at this time, so decided to remove it to keep things simpler.